Source/Utils/WarpXUtil.cpp

```diff
@@ -602,6 +602,34 @@ void CheckGriddingForRZSpectral ()
 #endif
 }
 
+void OverwriteAMReXperiodic ()
+{
+    // Check that `is_periodic` is not already set.
+    amrex::Vector<int> geom_periodicity(AMREX_SPACEDIM,0);
+    ParmParse pp_geometry("geometry");
+    if (pp_geometry.queryarr("is_periodic", geom_periodicity)) {
+        amrex::Abort("geometry.is_periodic is not supported. Please use `boundary.field_lo`, `boundary.field_hi` to specifiy field boundary conditions and 'boundary.particle_lo', 'boundary.particle_hi'  to specify particle boundary conditions.");
+    }
+
+    // Check which dimention of space is periodic, according to
+    // `boundary.field_lo`. (Consistency between `field_lo` and `field_hi`
+    // is checked in the function `ReadBCParams`)
+    amrex::Vector<std::string> field_BC_lo(AMREX_SPACEDIM,"default");
```

Comment:
A tiny detail - the might be easier to follow if the variable name is the same as the input parameter name, `field_lo`.